testsuite/policy/tls_policy.py

```diff
@@ -55,9 +53,5 @@ def __getitem__(self, key):
     def wait_for_ready(self):
         """TLSPolicy does not have Enforced
         https://github.com/Kuadrant/kuadrant-operator/issues/572"""
-        with oc.timeout(90):
-            success, _, _ = self.self_selector().until_all(
-                success_func=has_condition("Accepted", "True"),
-                tolerate_failures=5,
-            )
-            assert success, f"{self.kind()} did not get ready in time"
+        success = self.wait_until(has_condition("Accepted", "True"))
```

Comment:
TLSPolicy has been updated with `Enforced` recently: https://github.com/Kuadrant/kuadrant-operator/pull/635
so this whole method can be removed since the version from `super` should be sufficient.

Actually, I think that `wait_for_ready` can be implemented once in base `Policy` class now, there's no need to have specific version of this method for particular policies at all.

Reply:
Eh, there is that additional `time.sleep(5)` for RLP so RLP needs its own `wait_for_ready` method.